dashboard/src/pages/new.tsx

```diff
@@ -171,52 +177,37 @@ export function NewProjectPageContent({
       }
     }
 
-    // NOTE: Maybe we'll reintroduce this way of creating the subdomain in the future
-    // https://www.rfc-editor.org/rfc/rfc1034#section-3.1
-    // subdomain max length is 63 characters
-    // const subdomain = `${slug}-${workspaceSlug}`.substring(0, 63);
+    let projectSlug = '';
 
     try {
-      if (isK8SPostgresEnabledInCurrentEnvironment) {
-        await insertApp({
-          variables: {
-            app: {
-              name,
-              slug,
-              planId: plan.id,
-              workspaceId: selectedWorkspace.id,
-              regionId: selectedRegion.id,
-              postgresPassword: databasePassword,
-            },
-          },
-        });
-      } else {
-        await insertApp({
-          variables: {
-            app: {
-              name,
-              slug,
-              planId: plan.id,
-              workspaceId: selectedWorkspace.id,
-              regionId: selectedRegion.id,
-            },
-          },
-        });
-      }
+      const variables = {
+        name,
+        planId: plan.id,
+        workspaceId: selectedWorkspace.id,
+        regionId: selectedRegion.id,
+        postgresPassword: isK8SPostgresEnabledInCurrentEnvironment
+          ? databasePassword
+          : undefined,
+      } as CreateNewAppMutationVariables;
+
+      const { data } = await createNewApp({
+        variables,
```

Comment:
We should just pass the object directly to the mutation fn. We wouldn't need to explicitly type cast it